src/csync/CMakeLists.txt

```diff
@@ -19,7 +19,15 @@ include(DefineOptions.cmake)
 
 include(DefineInstallationPaths)
 
-find_package(SQLite3 3.8.0 REQUIRED)
+if (APPLE OR WIN32)
```

Comment:
Please keep only there the automatic choice for using the embedded SQLite copy here and then fill in all variables later.
So
```
if (APPLE OR WIN32)
    set(USE_OUR_OWN_SQLITE3 TRUE)
else()
    find_package(SQLite3 3.8.0 REQUIRED)
endif()
```

and later if `USE_OUR_OWN_SQLITE3` is true, set the needed variables

Reply:
I changed it and introduced and made sqlite it's own target, to get rid of the extra if branch.